libportability-gfx/src/lib.rs

```diff
@@ -4,10 +4,14 @@
 #![allow(improper_ctypes)] //TEMP: buggy Rustc FFI analysis
 
 extern crate gfx_hal as hal;
+#[cfg(feature = "metal")]
+extern crate gfx_backend_metal as back;
 #[cfg(feature = "dx12")]
 extern crate gfx_backend_dx12 as back;
 #[cfg(feature = "vulkan")]
 extern crate gfx_backend_vulkan as back;
+#[cfg(not(any(feature = "metal", feature = "dx12", feature = "vulkan")))]
```

Comment:
Why not just `#[cfg(feature = "metal")]`? Maybe the `empty` backend as default case?